libs/ui/src/components/NeoNftCard/NFTMediaInfo.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div
+    class="nft-media-info is-flex is-flex-direction-column"
+    :class="`nft-media-info__${variant}`">
+    <div class="is-flex is-flex-direction-column">
+      <span
+        class="is-ellipsis has-text-weight-bold"
+        data-testid="nft-name"
+        :title="nft.name"
```

Comment:
fix ts warning? since `nft?` is optional props

![image](https://github.com/kodadot/nft-gallery/assets/734428/179a7567-df7a-4774-a63d-b9305eb9e7e8)